.github/workflows/artifact_cleanup.yml

```diff
@@ -0,0 +1,17 @@
+# Our unittest action uses GitHub artifacts to store coverage files from the 
+# frontend and backend jobs for consumption by the coverage jobs. This cleans 
+# up old artifact files that we don't know once the coverage jobs have run.
+# https://github.com/marketplace/actions/purge-artifacts 
+name: 'Artifact cleanup'
```

Comment:
We don't need this job if we're not storing coverage files off between jobs right now.

Reply:
Yeah, never finished my commit to clean this up last week. Completing that now.